src/api.ts

```diff
@@ -21,4 +21,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId?: number) => get<User>(`/users/${userId}`);
```

Comment:
<img width="365" alt="image" src="https://user-images.githubusercontent.com/86183007/202998408-8e8d200d-8c0c-4669-a061-b9ffc74b26ec.png">

Reply:
As for me, something wrong🫠